flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1210,7 +1210,7 @@ def run_convergence_step(self, desired_datasets):
         note("Running changes: {changes}".format(changes=changes))
         self.successResultOf(run_state_change(changes, self.deployer))
 
-    def run_to_convergence(self, desired_datasets, max_iterations=4):
+    def run_to_convergence(self, desired_datasets, max_iterations=6):
```

Comment:
Why 6? If that's what it takes exactly, it might take more in future... and then we'll get mysterious failures. On presumption that iterations are cheap once convergence is achieved, I suggest picking a number that is definitely big enough, e.g. 20, and explaining why.